pkgs/development/tools/misc/act/default.nix

```diff
@@ -2,23 +2,24 @@
 
 buildGoModule rec {
   pname = "act";
-  version = "0.2.8";
+  version = "0.2.9";
 
   src = fetchFromGitHub {
     owner = "nektos";
     repo = pname;
     rev = "v${version}";
-    sha256 = "14ird8z8f467spa0kdzjf6lq7pipq7rwxrdk6ppv7y1fxw96qm9x";
+    sha256 = "17w7pqpn9pkzc85lrsmyhxy6rip47dxw1hkz4ml3y5n68nysfpb9";
   };
 
-  vendorSha256 = "0ns20vvrj0j921wsx227dxbpga6kll7pxglfqhl53xckrh85yyd8";
+  vendorSha256 = "0qf26g0a2j1mbzlc7xjackww22w9bl1x0iw3q1x6kq7fp8xiwhdn";
 
   buildFlagsArray = [ "-ldflags=-s -w -X main.version=${version}" ];
 
   meta = with lib; {
     description = "Run your GitHub Actions locally";
     homepage = "https://github.com/nektos/act";
+    changelog = "https://github.com/nektos/act/releases/tag/v${version}";
```

Comment:
@marsam Interesting. I saw `meta.changelog` mentioned in the manuals, but seldom saw it being used. Is there any policy/convention for adding this to projects (for examples, ones that have nice changelogs in their release descriptions), or is this just a matter of personal preference?

Reply:
AFAIK there is no policy, I usually add it to packages which I'm interested in their changelog. 
btw, congrats for becoming a committer, we're delighted to have you!

Reply:
As some extra context, the `r-ryantm` bot will now link to `meta.changelog` if it exists, like so:
https://github.com/NixOS/nixpkgs/pull/89682

So please do add these, as it saves reviewer time and is a nice quality-of-life improvement!